bazel/repository_locations.bzl

```diff
@@ -69,6 +69,14 @@ REPOSITORY_LOCATIONS = dict(
         commit = "d6f78d948c53f3b400bb46996eb3084359914f9b",
         remote = "https://github.com/google/googleapis",
     ),
+    com_github_datadog_dd_opentracing_cpp = dict(
```

Comment:
It'd be nice if they let us use tags, but I understand completely why they don't.